packages/react-cosmos-apollo-proxy/src/__tests__/client.js

```diff
@@ -114,21 +113,21 @@ describe('proxy configured with a client', () => {
   });
 
   beforeEach(() => {
-    client = new ApolloClient({
+    clientOptions = {
       cache: new InMemoryCache(),
       link: new HttpLink({ uri: 'https://xyz' })
-    });
+    };
 
-    setupTestWrapper({ proxyConfig: { client } });
+    setupTestWrapper({ proxyConfig: { clientOptions } });
   });
 
-  it('uses the client passed in the config', () => {
-    expect(wrapper.instance().client).toBe(client);
+  it('uses the clientOptions passed in the config', () => {
+    expect(wrapper.instance().client.cache).toBe(clientOptions.cache);
   });
 
-  it('connects to the Apollo DevTools', () => {
-    expect(parent.__APOLLO_CLIENT__).toBe(client);
-  });
+  //   it('connects to the Apollo DevTools', () => {
+  //     expect(parent.__APOLLO_CLIENT__).toBe(clientOptions);
```

Comment:
@JannesV Can we unbury this test? Maybe we can check that `parent.__APOLLO_CLIENT__` equals `wrapper.instance().client`.

Reply:
Seems like a good idea, changed ;)